test/python/transpiler/test_commutation_analysis.py

```diff
@@ -42,7 +42,7 @@ def assertCommutationSet(self, result, expected):
             for commutation_set in sets:
                 result_to_compare[qbit].append(sorted(node._node_id for node in commutation_set))
 
-        for qbit, sets in expected.items():
+        for _qbit, sets in expected.items():
```

Comment:
`for sets in expected.values()`

Reply:
Suggested change made in 3f35fdd52b95ea216d91594e3b1fd5eb7faecac2